decidim-generators/lib/decidim/generators/app_templates/initializer.rb

```diff
@@ -1,488 +0,0 @@
-# frozen_string_literal: true
-
-# most of this configurations can be handled through the environment variables.
-# Please refer to: https://docs.decidim.org/en/develop/configure/environment_variables
-
-Decidim.configure do |config|
-  # The name of the application
-  # config.application_name = Decidim::Env.new("DECIDIM_APPLICATION_NAME", "My Application Name").to_s
-
-  # The email that will be used as sender in all emails from Decidim
-  # config.mailer_sender = Decidim::Env.new("DECIDIM_MAILER_SENDER", "change-me@example.org").to_s
-
-  # Sets the list of available locales for the whole application.
-  #
-  # When an organization is created through the System area, system admins will
-  # be able to choose the available languages for that organization. That list
-  # of languages will be equal or a subset of the list in this file.
-  config.available_locales = (Decidim::Env.new("DECIDIM_AVAILABLE_LOCALES").presence || [:ca, :cs, :de, :en, :es, :eu, :fi, :fr, :it, :ja, :nl, :pl, :pt, :ro]).to_a
-
-  # Or block set it up manually and prevent ENV manipulation:
-  # config.available_locales = %w(en ca es)
-
-  # Sets the default locale for new organizations. When creating a new
-  # organization from the System area, system admins will be able to overwrite
-  # this value for that specific organization.
-  # config.default_locale = Decidim::Env.new("DECIDIM_DEFAULT_LOCALE", "en").to_s
-
-  # Restrict access to the system part with an authorized ip list.
-  # You can use a single ip like ("1.2.3.4"), or an ip subnet like ("1.2.3.4/24")
-  # You may specify multiple ip in an array ["1.2.3.4", "1.2.3.4/24"]
-  # config.system_accesslist_ips = Decidim::Env.new("DECIDIM_SYSTEM_ACCESSLIST_IPS").to_array
-
-  # Defines a list of custom content processors. They are used to parse and
-  # render specific tags inside some user-provided content. Check the docs for
-  # more info.
-  # config.content_processors = []
-
-  # Whether SSL should be enabled or not.
-  # if this var is not defined, it is decided automatically per-rails-environment
-
-  # Decidim::Env.new("DECIDIM_FORCE_SSL", "auto").default_or_present_if_exists
-  # config.force_ssl = Decidim::Env.new("DECIDIM_FORCE_SSL").present? unless Decidim::Env.new("DECIDIM_FORCE_SSL", "auto").default_or_present_if_exists.to_s == "auto"
-  # or set it up manually and prevent any ENV manipulation:
-  # config.force_ssl = true
-
-  # Enable the service worker. By default is disabled in development and enabled in the rest of environments
-  # config.service_worker_enabled = Decidim::Env.new("DECIDIM_SERVICE_WORKER_ENABLED", Rails.env.exclude?("development")).present?
-
-  # Sets the list of static pages' slugs that can include content blocks.
-  # By default is only enabled in the terms-of-service static page to allow a summary to be added and include
-  # sections with a two-pane view
-  # config.page_blocks = Decidim::Env.new("DECIDIM_PAGE_BLOCKS", "terms-of-service").to_array
```

Comment:
This isn't documented at docs/modules/configure/pages/environment_variables.adoc

Reply:
done in e36f7497ff29eee31ee0291b3ea5ed0239a24a8d